ros_buildfarm/common.py

```diff
@@ -42,6 +42,30 @@ def __init__(self, message):  # noqa: D107
         super(JobValidationError, self).__init__(message)
 
 
+class RepositoryPackageDescriptor(str):
+    """
+    Represents a package stored in a platform-specific package
+    repository.
+
+    Currently the class is inheriting from str for backwards compatibility.
+    You should not rely on this but use the `version` property instead.
+
+    To be replaced with:
+    namedtuple('RepositoryPackageDescriptor', 'name version')
+    """
+
+    @staticmethod
+    def __new__(cls, name, version):
+        return str.__new__(cls, version)
+
+    def __init__(self, name, version):
+        self.name = name
```

Comment:
Maybe remove `name` from the new class for now? It can be added later when it is actually being used.

Reply:
5c01b1b